rust/operator-binary/src/config.rs

```diff
@@ -0,0 +1,57 @@
+use stackable_superset_crd::authentication::{AuthenticationClass, AuthenticationClassType};
+
+pub fn compute_superset_config(authentication_class: &Option<AuthenticationClass>) -> String {
+    // We don't calculate the secrets here directly, as the operator should not be able to see the actual credentials.
+    // Instead we add a env-var which gets it's value from the secret with the credentials and read that with Python.
+    let common = r#"
+# common configs
```

Comment:
If you take a look at the current config.rs it now has a lot more format strings and i think i would make it difficult to read if we pull all of them out into separate files

Reply:
Yeah.. those are all currently completely unescaped (https://github.com/stackabletech/superset-operator/pull/145/files/e54c1406da7ca2eb76db0f1558750c8d3e67d656#r826035108). I'd much rather stick them in through a secondary channel (either env vars or a JSON file we deploy next to the config.py perhaps?) and deploy the same config.py everywhere which then reads that secondary channel.

Reply:
I would like to establish like a "common pattern" of how to do that in all the products supporting LDAP.
And not all products support executing arbitrary code to collect the Env/Json. Maybe we should a proper validation/escaping into the AuthenticationClass so that we validate all properties in a single place and the operators can behave all the same way? (I'm afraid we need proper validation/escaping anyhow)

Reply:
Hmm would be ok for me to put this into env vars and than read them with Python

Reply:
That should be fine.